app/views/downtimes/_form.html.erb

```diff
@@ -7,32 +7,32 @@
   year: {
     name: "downtime[start_time(1i)]",
     label: "Year",
-    value: params.dig("downtime", "start_time(1i)"),
+    value: @downtime.start_time&.year || params.dig("downtime", "start_time(1i)"),
```

Comment:
I'm wondering whether we should reverse the order of these since, as it currently is, when trying to set an invalid value (e.g. omitting a day value) and submitting, the page renders with a validation error message, but the value in the text field will be the previously valid value, not the submitted invalid value, which might be confusing for users.